frontend/app/routes/_gcweb-app.personal-information._index.tsx

```diff
@@ -55,18 +57,36 @@ export default function PersonalInformationIndex() {
         </PersonalInformationSection>
         <PersonalInformationSection
           footer={
-            <Link id="change-home-address-button" className="btn btn-primary btn-lg" to="/personal-information/address/edit">
-              {t('personal-information:index.change-addresses')}
+            <Link id="change-home-address-button" className="btn btn-primary btn-lg" to="/personal-information/home-address/edit">
+              {t('personal-information:index.change-home-address')}
+            </Link>
+          }
+          title={t('personal-information:index.home-address')}
+          icon="glyphicon-map-marker"
```

Comment:
**Massive Nitpick (feel free to ignore):** Consider using different icons for home address and mailing address.